### PR TITLE
Configure Civic UK branding and text 

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
Cookie work and ticket #10673 
[Designs can be found here](https://www.figma.com/file/N76hkJsMkSb2O6ToyQPxax/Cookie-journeys-update?node-id=1718%3A84945&mode=dev) but are more for inspiration than perfect copying in this context.

## What is it doing for them?
I'll add comments in the codebase to explain some interesting bits. This is still just a work in progress and is not the final version of what we'll end up with.

Taster screenshots (ignore the lack of Essential only button, it's a point to discuss):
Screenshot 2024-03-01 at 10 31 55
Screenshot 2024-03-01 at 10 32 02

### CODE REVIEW

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:7]: More options to explore here: https://www.civicuk.com/cookie-control/documentation/branding

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:34]: Config to be found here: https://www.civicuk.com/cookie-control/documentation/cookies#privacy-statement

Would be good to have a date on the page as well I reckon, and to know what date that is for our config; I don't think it can be found at the mo on the page?

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:50]: Allows us to write HTML in the text 🙌

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:55]: This and the line above is meant to work for both the notify banner and the preference centre; couldn't see how to split that preference, seems to be "display on both" or "hide on both" (but see previous line's comment)

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:60]: This is just a test; I believe we can only set one font family with their config, but as we can apply classes, we can cheat that. See with Jason.

File: common/views/pages/_app.tsx
Comment (rcantin-w) [inline common/views/pages/_app.tsx:87]: As in [mockups](https://www.figma.com/file/N76hkJsMkSb2O6ToyQPxax/Cookie-journeys-update?node-id=1718%3A84945&mode=dev), but review with Jason.

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:4]: Icon is removed, but decide with Jason where the cookie preference centre link should be and look like.

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:11]: Removes default, which we don't seem to want.

File: common/services/app/civic-uk/scripts.tsx
Comment (rcantin-w) [inline common/services/app/civic-uk/scripts.tsx:18]: This we need to discuss because the dismiss button isn't really in our plans but can't be removed from the actual preference centre, so we need to decide on copy. I made it a button so we have more control, but need to decide on that.

Comment (kenoir) [reply common/services/app/civic-uk/scripts.tsx:18]: I thought `notifyDismissButton` might do this, but it relies on being in the `notify` initial state?

Screenshot 2024-03-01 at 10 43 40

Comment (rcantin-w) [reply common/services/app/civic-uk/scripts.tsx:18]: That's just for the banner, which I set to false as I think we want to force a click on a button as much as possible. The `notify` prefix is used to configurate that first state (like `notifyTitle` or `notifyDescription`)

File: common/services/app/civic-uk/scripts.tsx
Comment (kenoir) [reply common/services/app/civic-uk/scripts.tsx:33]: Not directly related to this PR, but should we have our own privacy statement page on `wellcomecollection.org`? We'll have different cookies from `wellcome.org` and being able to manage that list ourself would be useful.

Comment (rcantin-w) [reply common/services/app/civic-uk/scripts.tsx:33]: I think I remember we did say it would be the way we'd go and in that case, we want to make sure we have a date there (that I think will just be part of the copy as it shouldn't change based on publication if someone republishes to fix a typo). Or I wonder if we could fetch that date from ?somewhere? and display it in both places automatically, that'd be ideal.

File: common/views/pages/_app.tsx
Comment (rcantin-w) [reply common/views/pages/_app.tsx:87]: Remove this

**Review by kenoir — COMMENTED** [review]
Are we looking to merge this? I think if it's behind a toggle it's pretty safe. Looks like there is a tiny bit of clean up to do, but otherwise good to go.

**Comment by rcantin-w** [comment]
> Are we looking to merge this? 

Yess just waiting for the usual review!

> Looks like there is a tiny bit of clean up to do

which bits? There is also https://github.com/wellcomecollection/wellcomecollection.org/pull/10685 following this one, but wanted to just get this one in to start with, split the review work.

**Review by kenoir — APPROVED** [review]
Think there is a comment to remove, but all good to go other than that.

**Comment by rcantin-w** [comment]
Ahhh yes, [this one](https://github.com/wellcomecollection/wellcomecollection.org/pull/10681/files#r1512571548)! 

Should've been more descriptive, it was about not doing that at all ("Have popup that says "Your cookie settings have been saved""), so I've removed it [here in the next PR](https://github.com/wellcomecollection/wellcomecollection.org/pull/10685/files#diff-3ad843d4162d6fbe90f9d61ddd27862b9271962f05dc1d842689952cba89b2d8R9). I'll merge this one in 👍

**Pull request merged**
